addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -5318,6 +5291,9 @@ registry.layout_column = SnippetOptionWidget.extend({
         // Removing the grid class
         rowEl.classList.remove('o_grid_mode');
         const columnEls = rowEl.children;
+        // Removing the grid previews (if any).
+        this._clearAllPreviews(columnEls);
```

Comment:
Why is this necessary? Should not your `cleanUI` method be called at the right moment to handle this automatically?

Reply:
It is for the eventual case the user would toggle the normal mode with some previews still displayed, so the grid preview will not be considered as a child and the padding highlight classes will not be on the columns (really a corner case I think but there are tracebacks if he does that so better be safe...)  :grimacing: 

You are right, I could just call `cleanUI` there then :slightly_smiling_face: I added this function because I didn't know how to call the one of the GridColumn option but as you mentioned in an another comment, I can make them communicate with `notify` indeed :open_mouth: 
So yes the next comment will surely be fixed by that :slightly_smiling_face:  (I didn't know how to do that in another way which is why storing in the DOM was the solution I used but I knew it would not be accepted haha)

Reply:
In the end, I trigger a `clean_ui_request`, because it seems that the `notify` only works for options of the same editor and not from an editor to another (so here I can't communicate between a "section" option and a "column" option) 
Or did I miss something ?